ansible/postgresql-migrate.yaml

```diff
@@ -0,0 +1,9 @@
+- hosts: localhost
+  vars:
```

Comment:
Please move all these vars as a sibling of `role` Also add a tag called `sunbird-programs` for the role. In future we can keep appending new dbs by adding additional role tags